editor.html

```diff
@@ -259,7 +259,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                             </ul>
                         </div>
                         <div class="buttons_menu_item">
-                            Select Language:
+                            <p id="lang-select">Select Language:</p>
```

Comment:
Using a `p` elements adds whitespace around this text, messing with the tree view spacing:
![image](https://user-images.githubusercontent.com/29712657/61953939-8d85bb00-afaf-11e9-93ce-a0f43004aa7e.png)

Instead you can use the `span` element.

Reply:
Yep, I'm unsure why I didn't use span when I used it on the  other help dropdown elements